### DISCUSSION TITLE
【玩家招募】我的世界JAVA1.21原版生存服务器

### DISCUSSION THREAD

**Discussion by H-luoyan** [share121/inter-knot#2639] [其他]
欢迎大家加入我的服务器，连接地址111.161.122.206:49170，也欢迎大家进群交流

![0d2cd3de86f67509dd20193050cd8b7](https://github.com/user-attachments/assets/33ccc921-b7b7-49f7-ad28-1d4f87a39b2b)

**Comment by yytyfj** [comment]
1.21改了好多东西，偏萌新要吗<img src="https://bbs-static.miyoushe.com/static/2023/04/28/0c624a8915dd9b24093cf630bded5748_7288657913684710277.png" alt="米仔_拜年" width="75" >

**Reply by H-luoyan** [reply]
萌新也可以的，新加的东西可以以后再探索<img src="https://bbs-static.miyoushe.com/static/2024/07/31/fdc5e0e30e577fb208f29effa93510ed_8442612635207222552.png" alt="云璃 期待" width="75" >

**Reply by yytyfj** [reply]
好欸

**Comment by jsishsjsj** [comment]
建议将ip去掉，只保留QQ群，这样可以防止被D（虽然不能避免完全被D，但还是可以避免一些）<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >

**Comment by Momsaysnamesthataretoolongmakefoolslook** [comment]
需要正版吗
